src/nft/components/explore/CarouselCard.tsx

```diff
@@ -295,13 +296,12 @@ const CollectionName = styled(ThemedText.MediumHeader)`
 `
 
 const CarouselCardHeader = ({ collection }: { collection: TrendingCollection }) => {
-  const theme = useTheme()
   return (
     <CardHeaderContainer src={collection.bannerImageUrl}>
       <CardHeaderColumn>
         <CollectionImage src={collection.imageUrl} />
         <CollectionNameContainer>
-          <CollectionName color={theme.accentTextLightPrimary} fontWeight="500">
+          <CollectionName color="accentTextLightPrimary" fontWeight="500">
```

Comment:
could you just add these styles to the styled component?

Reply:
It's a `ThemedText.MediumHeader`, so I wanted to stick to the way it does colors/fonts until we come up with better abstraction (if any - just have some general thoughts on this component, but it's not related to this PR)